roles/sensu/server/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+#Add the following configuration
+# api.json -- it configures the sensu-api (server and client)
+# redis.json -- it configures the use of redis for sensu as storage
+# sensu_oscheck.json -- it configures the oschecks to be executed on the clients
+#    sensu_oschecks.json.j2 is a template with severals templates that can be 
+#    disabled and configured 
+- name: configure sensu
+  template:
+   src: 'templates/{{ item }}.json.j2'
+   dest: '{{sensu_config_path}}/{{ item }}.json'
+   owner: '{{ sensu_owner }}'
+   group: '{{ sensu_group }}'
+  with_items:
+      - api
+      - redis
+      - sensu_oschecks
+  notify:
+      - restart sensu
+
+#This task is needed due to the fact that all the templates used to 
+#create sensu_oschecks.json can be disabled. 
+#All these templates end with a coma ','
+#The last coma ',' needs to be deleted to not let sensu-server crash
+- name: fix sensu_oschecks
+  replace:
+   dest: '{{sensu_config_path}}/sensu_oschecks.json'
+   regexp: ",\n +}}"
+   replace: "\n    }\n}"
+
+#Include the rabbitmq plugin to be able to create a vhost
+- name: Add plugin to manage rabbitmq
+  rabbitmq_plugin:
+   names: rabbitmq_management
+   state: enabled
+
+#It creates the vhost to be used
+- name: Create sensu vhost on rabbitmq
+  rabbitmq_vhost:
+   name: '{{ sensu_rabbitmq_vhost }}'
+   state: present
+  notify:
+      - restart rabbitmq
+
+# Add user to server and assign full access control on / vhost.
+# The user doesn't have permission rules for other vhosts
+- name: Configure rabbitmq permissions
+  rabbitmq_user:
+    user: '{{ sensu_rabbitmq_user }}'
+    password: '{{ sensu_rabbitmq_password }}'
+    vhost: '{{ sensu_rabbitmq_vhost }}'
+    configure_priv: '.*'
+    read_priv: '.*'
+    write_priv: '.*'
+  notify:
+      - restart rabbitmq
```

Comment:
Let me explain, at the beginning it was as you suggested.
So I changed because because another suggestion, and at that point I was convinced.
Saying that, I agree with you. It shouldn't be here; it might be in the rabbitmq/sensu???